### DISCUSSION TITLE
Googling "fastapi httpbasic" can lead you to emoji (/em/) version of the documentation page 😕

### DISCUSSION THREAD

**Discussion by przecze** [fastapi/fastapi#11505] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
not applicable
```

### Description

* Search for `fastapi httpbasic`
<img width="669" alt="image" src="https://github.com/tiangolo/fastapi/assets/25085890/37110494-7ad8-4de8-8600-a41a0ff9549a">

* Click on the second link
* you see https://fastapi.tiangolo.com/em/advanced/security/http-basic-auth/ which is the emoji version of the page
<img width="673" alt="image" src="https://github.com/tiangolo/fastapi/assets/25085890/f3e48eea-37aa-450d-9a23-cea9e4a61f95">

Honestly before I noticed `/em/` in the path which looked like language identifier I was very confused. It looked like the website got hacked which was concering especially as I was researching a security feature.
Maybe the `/em/` version can be disabled from search engines indexing to avoid confusing situations like that?

### Operating System

macOS

### Operating System Details

not applicable

### FastAPI Version

in top right I see doc page renders for v 0.110.3

### Pydantic Version

not applicable

### Python Version

not applicable

### Additional Context

I'm located in Germany
The problem occures both when I'm logged in to chrome with preferred language set to english in google account, and when I use the chrome anonymouos profile with preferred language set to english in chrome settings
<img width="724" alt="image" src="https://github.com/tiangolo/fastapi/assets/25085890/037bf930-dcb1-478a-8d2d-b8ce221d6c45">
<img width="655" alt="image" src="https://github.com/tiangolo/fastapi/assets/25085890/4eebf32a-9641-4282-8fba-a8cd5f237fda">

**Comment by rubzip** [comment]
Same to me! I searched in google: "httpexception fastapi" entered to https://fastapi.tiangolo.com/em/tutorial/handling-errors/ and the result was a page full of emojis, an extract of the content:
🚚 ❌[¶](https://fastapi.tiangolo.com/em/tutorial/handling-errors/#_1)
📤 📚 ⚠ 🌐❔ 👆 💪 🚨 ❌ 👩‍💻 👈 ⚙️ 👆 🛠️.

👉 👩‍💻 💪 🖥 ⏮️ 🕸, 📟 ⚪️➡️ 👱 🙆, ☁ 📳, ♒️.

👆 💪 💪 💬 👩‍💻 👈:

👩‍💻 🚫 ✔️ 🥃 😌 👈 🛠️.
👩‍💻 🚫 ✔️ 🔐 👈 ℹ.
🏬 👩‍💻 🔄 🔐 🚫 🔀.
♒️.
👫 💼, 👆 🔜 🛎 📨 🇺🇸🔍 👔 📟 ↔ 4️⃣0️⃣0️⃣ (⚪️➡️ 4️⃣0️⃣0️⃣ 4️⃣9️⃣9️⃣).

👉 🎏 2️⃣0️⃣0️⃣ 🇺🇸🔍 👔 📟 (⚪️➡️ 2️⃣0️⃣0️⃣ 2️⃣9️⃣9️⃣). 👈 "2️⃣0️⃣0️⃣" 👔 📟 ⛓ 👈 😫 📤 "🏆" 📨.

👔 📟 4️⃣0️⃣0️⃣ ↔ ⛓ 👈 📤 ❌ ⚪️➡️ 👩‍💻.

💭 🌐 👈 "4️⃣0️⃣4️⃣ 🚫 🔎" ❌ (& 🤣) ❓
